extensions/amp-analytics/0.1/url-replacement-whitelists.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Used for inserted scoped analytics element.
+ * @const {!Object<string, boolean>}
+ */
+export const ScopedElementWhiteList = {
+  // TODO: Please Review the list!
+  'RANDOM': true,
+  'COUNTER': true,
+  'CANONICAL_URL': false,
```

Comment:
I assume all `CANONICAL_URL`, `CANONICAL_HOST`, `CANONICAL_HOSTNAME`, `CANONICAL_PATH` are good. I set all of them to true.